google-cloud-clients/google-cloud-spanner/pom.xml

```diff
@@ -2,212 +2,242 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
```

Comment:
I don't think you are changing anything apart from tabs/space in pom.xml, Please revert changes in this file.

Reply:
I have added following dependency which require to create a RecordEventsSpanImpl Object.
```java
<dependency>
	<groupId>io.opencensus</groupId>
	<artifactId>opencensus-impl-core</artifactId>
	<version>0.24.0</version>
	<scope>compile</scope>
</dependency>
<dependency>
            <groupId>io.opencensus</groupId>
            <artifactId>opencensus-testing</artifactId>
            <version>0.24.0</version>
            <scope>compile</scope>
            <optional>true</optional>
</dependency>
```